scheduler/src/cook/kubernetes/controller.clj

```diff
@@ -42,21 +43,21 @@
   expected-state-dict)
 
 (defn update-or-delete!
-  [map-atom key value]
+  [^IAtom map-atom key value]
```

Comment:
Do we need the IAtom type hint?

Reply:
No, but it makes things more self-documenting. :)